pkg/controller/installation/core_controller.go

```diff
@@ -1626,6 +1627,20 @@ func getOrCreateTyphaNodeTLSConfig(cli client.Client, certificateManager certifi
 func (r *ReconcileInstallation) setDefaultsOnFelixConfiguration(ctx context.Context, install *operator.Installation, fc *crdv1.FelixConfiguration, reqLogger logr.Logger) (bool, error) {
 	updated := false
 
+	if install.Spec.CalicoNetwork.LinuxDataplane != nil {
+		if *install.Spec.CalicoNetwork.LinuxDataplane == operatorv1.LinuxDataplaneNftables && fc.Spec.NFTablesMode == nil {
```

Comment:
If I'm reading this right we will only set this to enabled if FelixConfig has NFTablesMode nil but we'll set disabled even if FelixConfig has NFTablesMode set.
That doesn't seem like symmetric behavior to me.

Reply:
Yeah you're absolutely right.